ops/terraform/env/mgmt/stateless/variables.tf

```diff
@@ -8,17 +8,18 @@ variable "jenkins_ami" {
   type              = string
 }
 
-variable "vpn_security_group_id" {
-  description       = "Security group that provides access via VPN"
-  type              = string
-}
-
 variable "jenkins_key_name" {
   description       = "The EC2 key pair name to assign to jenkins instances"
   type              = string
 }
 
-variable "instance_size" {
-  description       = "Size of EC2 Instance"
+variable "jenkins_instance_size" {
   type              = string
+  description       = "The EC2 instance size to use"
+  default           = "c5.xlarge"
```

Comment:
Changing from a General Purpose to a Compute Optimized is intentional, I assume?

Reply:
Yeah that is part of the app build optimization to reduce deploy times: https://jira.cms.gov/browse/BLUEBUTTON-1579